README.md

```diff
@@ -70,6 +76,12 @@ If Gatekeeper is no longer running, it has no ability to clean up after itself,
    * Executing `kubectl patch  crd constrainttemplates.templates.gatekeeper.sh -p '{"metadata":{"finalizers":[]}}' --type=merge`. Note that this will remove all finalizers on every CRD. If this is not something you want to do, the finalizers must be removed individually.
    * Delete the `CRD` and `ConstraintTemplate` resources associated with the unwanted constraint.
 
+If you deployed Gatekeeper using a single yaml from the [Installation](#Installation) section, then after you have completed the [Cleaning Up Old Constraints](#cleaning-up-old-constraints) section, you can delete all the Gatekeeper components with the following command:
```

Comment:
This would still leave the artifacts specified in "Cleaning Up Old Constraints"

We should move the instructions above that section accordingly.

Reply:
> This would still leave the artifacts specified in "Cleaning Up Old Constraints"

This step should be performed _after_ users have done with Cleaning Up Old Constraints. What resources are you referring to that would not be removed?

Reply:
If GK is not running, at a minimum:

   * Constraints
   * ConstraintTemplates
   * Constraint CRDs

It's fine if you want to suggest users clean up constraints if they want them removed. If that's the case, we should streamline the doc and merge the two uninstall sections where possible e.g.:

 * "Before uninstalling be sure to..."
 * "If you used `make` then..."
 * "If you used an image then..."
 * "If there are extra constraints then..."

Basically assume people are reading this section as a step-by-step guide.